app.json

```diff
@@ -4,38 +4,8 @@
     "postdeploy": "rake db:migrate ; rake db:seed"
   },
   "env": {
-    "DEVISE_SECRET_KEY": {
-      "required": true
-    },
-    "ES_HOST": {
-      "required": true
-    },
-    "FIND_URL": {
-      "required": true
-    },
-    "HTTP_PASSWORD": {
-      "required": true
-    },
-    "HTTP_USERNAME": {
-      "required": true
-    },
-    "LANG": {
-      "required": true
-    },
-    "RACK_ENV": {
-      "required": true
-    },
-    "RAILS_ENV": {
-      "required": true
-    },
-    "RAILS_LOG_TO_STDOUT": {
-      "required": true
-    },
-    "RAILS_SERVE_STATIC_FILES": {
-      "required": true
-    },
-    "SECRET_KEY_BASE": {
```

Comment:
Should this be:

```json
"SECRET_KEY_BASE": {
  "description": "Cryptographically secure secret key (typically used to generate a secret for cookie sessions)",
  "generator": "secret"
},
```

Reply:
It's populated as part of the VCAP_SERVICES variable, which does helpfully make it similar to staging but also needs a cleanup. This is blocking me from the signon work, though, and I think it would take to much time to properly refactor it now.